src/test/java/uk/gov/pay/connector/gateway/worldpay/WorldpayOrderRequestBuilderTest.java

```diff
@@ -311,6 +312,67 @@ public void shouldGenerateValidAuthoriseGooglePay3DSOrderRequestWithoutIpAddress
         assertEquals(OrderRequestType.AUTHORISE_GOOGLE_PAY, actualRequest.getOrderRequestType());
     }
 
+    @Test
+    public void shouldGenerateValidAuthoriseGooglePay3DSOrderRequestWithIpAddress() throws Exception {
```

Comment:
Can we use `3ds` instead of 3DS for consistency?

```suggestion
    public void shouldGenerateValidAuthoriseGooglePay3dsOrderRequestWithIpAddress() throws Exception {
```

Reply:
Done.